src/components/common/Toolbar/Toolbar.tsx

```diff
@@ -62,53 +75,62 @@ export function Toolbar({ children, ...props }: ToolbarProps) {
     [focusedToolbarKey]
   );
 
-  const handleKeyDown = useCallback((event: KeyboardEvent) => {
-    const list = ref.current;
-    const currentFocus = document.activeElement;
+  const handleKeyDown = useCallback(
```

Comment:
There are some issues when I compare this to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/toolbar_role:

1. A vertical toolbar should use the up/down buttons but not the left/right buttons.
2. A vertical toolbar has `aria-orientation="vertical"` set. (This can also be used for the styling).

We should also have the same tests for the vertical toolbar that we also already have for the horizontal toolbar.